### DISCUSSION TITLE
Does LightningLite still support various callbacks?

### DISCUSSION THREAD

**Discussion by marsggbo** [Lightning-AI/pytorch-lightning#11260] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
`LightningModule` can be coupled with various callbacks? I wonder if it is possible `LightningLite` can also reuse those callbacks, such as `wandb`, `checkpoint`?

**Comment by rohitgr7** [comment, marked as answer]
I don't think they are because the hooks in callbacks are just some code injections that are supposed to run at a certain point where lightning defines the training loops/validation loops but in `LightningLite` user defines the training loop, so it won't be possible. Although in the case of wandb for eg you can use their native API to log the stuff.
